### DISCUSSION TITLE
[Feature] Option to change sorting of photos per album

### DISCUSSION THREAD

**Discussion by arkus7** [immich-app/immich#7031] [Feature Request]
### The feature

There are some albums I'm happy with the default sorting of photos (last created/modified descending = newest first) but for most albums, I'd like to view them in the same order as they were created (in ascending order). 

Currently, there is no option to set the order of photos on a per-album basis.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by pakosalze** [comment]
+1 for me

**Comment by TivoRanger** [comment]
Seconded.  I am trying to use Immich to replace the functionality in Picasa.  Picasa had the ability to arbitrarily order photos within albums.  This allowed me to "tell a story" with order.  Current ordering is by date modified.  This would be analogous to iPhone album sort "custom order".  I am new to Immich but I would think this would be a base requirement for albums, giving their intended use.

**Comment by bo0tzz** [comment]
#1689
